client/src/components/Gallery.jsx

```diff
@@ -1,98 +1,80 @@
 import React from 'react';
 import styled from 'styled-components';
 
+import Modal from 'react-bootstrap/Modal';
+import ModalBody from 'react-bootstrap/ModalBody';
+import ModalDialog from 'react-bootstrap/ModalDialog';
+
 import { Container, Row, Col, Image } from 'react-bootstrap'
 
+import Carousel from './Carousel.jsx';
+
+
 class Gallery extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-
+      carModalIsOpen: false,
+      caroImage: ''
     }
   }
 
-  // layout(images, min, max) {
-  //   let result = [[images[0]]];
-  //   let random;
-  //   let i = 1;
-
-  //   while (i < images.length) {
-  //     random = Math.floor(Math.random() * (max - min)) + min;
-  //     result.push(images.slice(i, i + random));
-  //     i += random;
-  //   }
-
-  //   if (result.some(j => j.length < min)) {
-  //     return layout(images, min, max);
-  //   } else {
-  //     return result;
-  //   }
-  // }
-
+  handleClick(e) {

```

Comment:
consider making this name more specific to what it's doing

Reply:
Good idea! Will do.